compose/material3/material3/src/skikoMain/kotlin/androidx/compose/material3/ExposedDropdownMenu.skiko.kt

```diff
@@ -87,24 +81,24 @@ actual fun ExposedDropdownMenuBox(
 ) {
     val density = LocalDensity.current
     val windowInfo = LocalWindowInfo.current
-    var width by remember { mutableIntStateOf(0) }
-    var menuHeight by remember { mutableIntStateOf(0) }
+    var anchorWidth by remember { mutableIntStateOf(0) }
+    var menuMaxHeight by remember { mutableIntStateOf(0) }
     val verticalMarginInPx = with(density) { MenuVerticalMargin.roundToPx() }
 
     val focusRequester = remember { FocusRequester() }
     val expandedDescription = getString(Strings.MenuExpanded)
     val collapsedDescription = getString(Strings.MenuCollapsed)
 
-    val scope = remember(expanded, onExpandedChange, density, menuHeight, width) {
+    val scope = remember(expanded, onExpandedChange, windowInfo, density) {
```

Comment:
Shouldn't we depend on `containerSize` instead of just `windowInfo`?

Reply:
Hmm, good point, I'll fix it.

Reply:
On second thought, no, that's not a good idea. It means the EDM will recompose whenever the size of the window changes.

It works the way it is now, because `onGloballyPositioned` is called when the window is resized, and `menuMaxHeight` is updated, but doesn't cause a recomposition.